compiler/src/dmd/expressionsem.d

```diff
@@ -5677,7 +5677,8 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
             symtab = sds.symtab;
         }
         assert(symtab);
-        Identifier id = Identifier.generateIdWithLoc(s, exp.loc);
+        import std.string : fromStringz;
+        Identifier id = Identifier.generateIdWithLoc(s, exp.loc, cast(string) fromStringz(sc.parent.toPrettyChars()));
```

Comment:
```suggestion
        Identifier id = Identifier.generateIdWithLoc(s, exp.loc, cast(string) toDString(sc.parent.toPrettyChars()));
```

dmd may not depend on Phobos

Reply:
Good point! I'm pretty sure I've seen `fromStringz` in the code though... `ag` points at `dmd/frontend.d`...

But certainly that's not a good reason to add more instances. Thanks for your suggestion, applied!